src/xmp-parser.js

```diff
@@ -13,7 +13,6 @@ const tagTypes = {
     // DJI
     'drone-dji:CamReverse': 'bool',
     'drone-dji:GimbalReverse': 'bool',
-    'drone-dji:RtkFlag': 'bool',
```

Comment:
no need to put the numbers here? :)

Reply:
Indeed, it's one of the cases in `parseValue`